src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFUtility.java

```diff
@@ -141,6 +142,20 @@ public static void addRightLink(PDDocument document, PDPage from, PDPage to, Str
         addLink(document, from, to, text, yyOffset, pageWidth - stringWidth - 53, font, fontSize);
     }
 
+    public static void verifyCoordinates(Bundle bundle) {
```

Comment:
I don't think this class has any knowledge of a Bundle - perhaps this should be a method on the bundle itself?